src/__tests__/fieldName.spec.ts

```diff
@@ -1,31 +0,0 @@
-import { fieldName } from '..';
-
-describe('formtron/fieldName', () => {
-  test('plain', () => {
-    const schema = {
-      type: 'string',
-    };
-    const field = fieldName(schema);
-    expect(field).toEqual('string');
-  });
-  test('array', () => {
-    const schema = {
-      type: 'array',
-      items: {
-        type: 'string',
-      },
-    };
-    const field = fieldName(schema);
-    expect(field).toEqual('string[]');
-  });
-  test('object', () => {
-    const schema = {
-      type: 'object',
-      items: {
-        type: 'string',
-      },
-    };
-    const field = fieldName(schema);
-    expect(field).toEqual('string{}');
-  });
```

Comment:
Why has this test been completely removed?

Reply:
Because `fieldName` the function was a stupid hack and will be completely removed.

Reply:
Nvm I get what you mean now